recipes/qvina/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "qvina" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hadim/qvina/archive/cmake.zip
+  sha256: f797b96fef7fc33667484aaf242c4d2d98c81525871d7959c654e429e42f9701
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake >=3.12
+    - make
+    - boost >=1.71.0
```

Comment:
boost should be in the host requirements section because it is linked against, and you should use `boost-cpp` to avoid the full boost library with Boost::Python